tests/src/test/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleServerSuite.scala

```diff
@@ -120,18 +121,21 @@ class RapidsShuffleServerSuite extends RapidsShuffleTestHelper {
       bounceBuffer
     }
     assertResult(true)(bb.deviceBounceBuffer.isClosed)
+    assertResult(true)(transferRequest.isClosed)
```

Comment:
```suggestion
    assert(transferRequest.isClosed)
```
Applies elsewhere in this file.